src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ReleaseDateService.kt

```diff
@@ -42,6 +42,13 @@ class ReleaseDateService(
     return actualReleaseDate < 1.workingDaysBefore(conditionalReleaseDate)
   }
 
+  fun isDueToBeReleasedInTheNextTwoWorkingDays(sentenceDateHolder: SentenceDateHolder): Boolean {
+    val actualReleaseDate = sentenceDateHolder.actualReleaseDate
+    val conditionalReleaseDate = sentenceDateHolder.conditionalReleaseDate
+    val earliestReleaseDate = listOfNotNull(actualReleaseDate, conditionalReleaseDate).minOrNull() ?: return false
+    return LocalDate.now(clock) > 2.workingDaysBefore(earliestReleaseDate) && LocalDate.now(clock) <= earliestReleaseDate
```

Comment:
Should this be `LocalDate.now(clock) >= 2.workingDaysBefore(earliestReleaseDate)`? Or is it like JS where `LocalDate.now() > LocalDate.now().startOfDay()` is true?

Reply:
LocalDate doesn't have a time component so `LocalDate.now() > LocalDate.now().startOfDay()` is not true.

but I rewrote the tests and yes it should have been `>=`